openid-4-verifiable-presentations-1_0.md

```diff
@@ -234,7 +234,7 @@ The Authorization Request follows the definition given in [@!RFC6749] taking int
 
 The Verifier may send an Authorization Request as Request Object by value or by reference as defined in JWT-Secured Authorization Request (JAR) [@RFC9101].
 
-The Verifier articulates requirements of the Credential(s) that are requested using `presentation_definition` and `presentation_definition_uri` parameters that contain a Presentation Definition JSON object as defined in Section 5 of [@!DIF.PresentationExchange]. Wallet implementations MUST process Presentation Definition JSON object and select candidate Verifiable Credential(s) using the evaluation process described in Section 8 of [@!DIF.PresentationExchange].
+The Verifier articulates requirements of the Credential(s) that are requested using `presentation_definition` and `presentation_definition_uri` parameters that contain a Presentation Definition JSON object as defined in Section 5 of [@!DIF.PresentationExchange]. Wallet implementations MUST process Presentation Definition JSON object and select candidate Verifiable Credential(s) using the evaluation process described in Section 8 of [@!DIF.PresentationExchange] unless implementing only a credential profile that provides rules on how to evaluate and process [@!DIF.PresentationExchange].
```

Comment:
MUST should now be relaxed to SHOULD, because the additional text says when the MUST does not apply

Reply:
> MUST should now be relaxed to SHOULD, because the additional text says when the MUST does not apply

The fact that it's a conditional must doesn't mean it needs to be downgraded to only a recommendation ('should') when the exception doesn't apply. I believe it should stay as a MUST.

Reply:
+1 to Joseph. the goal here is to say that when there is no credential profile, use as defined in PE, if there is (like 18013-7), use rules defined there.

Reply:
@David-Chadwick does that work for you?

Reply:
Yes sure